openslides/core/views.py

```diff
@@ -272,6 +272,10 @@ def prune_elements(self, request, pk):
             raise ValidationError({'detail': 'Data must be a list.'})
 
         projector_instance = self.get_object()
+        # reset scroll level
+        if (projector_instance.scroll != 0):
```

Comment:
The scroll level reset should be done after line 278 (serialiver.save()) because we do not want to reset something in case of ValidationError or other exception.